microstream-cache/build.gradle.kts

```diff
@@ -5,7 +5,11 @@ plugins {
 dependencies {
     api(project(":microstream"))
     api(libs.managed.microstream.cache)
-    api(mn.micronaut.cache.core)
+    api(mn.micronaut.cache.core) {
+        version {
+            strictly("4.0.0-SNAPSHOT")
+        }
+    }
```

Comment:
This can be removed once https://github.com/micronaut-projects/micronaut-core/pull/8275 is published